internal/home/home.go

```diff
@@ -763,6 +763,15 @@ func initWorkingDir(opts options) (err error) {
 	if opts.workDir != "" {
 		// If there is a custom config file, use it's directory as our working dir
 		Context.workDir = opts.workDir
+	} else if !execDirAvaliable() {
+		// If running as a service and from /usr/bin/ use /var/lib for working dir instead of
+		// /usr/bin/data
+		Context.workDir = "/var/lib/AdGuardHome"
+
+		// Create dir if it does not already exist
+		if err := os.MkdirAll(Context.workDir, 0755); err != nil {
```

Comment:
Here and in a few other places, prefer to not shadow the `err`.  See `make go-lint`.